### DISCUSSION TITLE
How is Employee Attendance Management implemented?

### DISCUSSION THREAD

**Discussion by arladmin** [exadel-inc/CompreFace#409] [Q&A]
Hi, i saw in your website that one of the use-cases is 'Employee Attendance Management'.

Is there any example code for this?

Does it also imply that this feature can be implemented off a continuous stream of (cctv/webcam) video (as that will be the only scenario for the execution of a facial-recognition attendance system in an office/warehouse) ?

Thanks.

**Comment by pospielov** [comment]
Exadel has experience with developing strangers recognition in the office, which is almost the same problem as Attendance Management, but unfortunately, I can't share the code (and that was a big project, it's not easy to find what code to share).
The architecture was like this:
1. The hardware camera that exposed video stream
2. A server for handling video
3. CompreFace server

So the server for handling video:
1. Reads the video stream and splits it into the frames
2. Finds the frames with motion
3. Sends them to CompreFace. 
4. Checks that on the consecutive frames are the same predicted person - for better accuracy.
